pcweb/pages/faq.py

```diff
@@ -0,0 +1,135 @@
+        ),
+    },
+    {
+        "Q": "What can I build with Reflex?",
+        "A": rx.vstack(
+            rx.text(
+            """
+            With Reflex, data scientists and software engineers can create high-quality web applications quickly and easily without needing to learn specific web development technologies. Whether you want to build a small-scale data science project/internal app, or a large-scale multi-page web app, Reflex has the tools and features to handle any project. 
+            """),
+            rx.text(
+                "Check out our ",
+                rx.span(doclink("gallery", href="/docs/gallery")),
+                " to see what you can build with Reflex.",
+            ),
+            align_items="flex-start",
+            width="100%",
+        )
+    },
+    {
+        "Q": "Whats the status on hosting?",
+        "A": rx.text(
+            """
+            Our hosting service is being actively developed and is aimed to be released Fall 2023. In the meantime, you can deploy your Reflex app to any hosting service following our 
+            """,
+            rx.span(doclink("self-hosting guide", href="/docs/hosting/self-hosting")),
+            ".",
+        ),
+    },
+    {
+        "Q": "How can I contribute?",
+        "A": rx.text(
+            """
+            We're always looking for contributors to help us build Reflex. 
+            If you're interested in contributing, check out our 
+            """,
+            rx.span(doclink("contributor program", href=constants.CONTRIBUTOR_PROGRAM_URL)),
+            "."
+        ),
+    },
+    {
+        "Q": "How does this compare existing solutions?",
+        "A": rx.vstack(
+            rx.text("""
+            Web apps are the most common way for developers to share their ideas. But even for skilled engineers, without experience in traditional frontend tools like Javascript or React, making a web app can be overwhelming and time-consuming. And once their app is created, deploying it is often a nightmare. Over the years, many low-code and no-code frameworks have tried to make web development more accessible, but they all have limitations and graduation risks compared to standard web frameworks. With Reflex, we have created a framework that lets developers leverage their existing Python skills to build and deploy apps without compromising on flexibility or customization.
+            """),
+            rx.text("""
+            Reflex allows you to make anything from a small data science project to a full-scale, multi-page web app. Since Reflex apps compile down to traditional frontend frameworks, there's no constraint on the type of apps you can build. Instead of reinventing the web development ecosystem, we're trying to make the existing ecosystem more accessible.
```

Comment:
```suggestion
            Reflex allows you to make anything from a small data science project to a full-scale, multi-page web app. Since Reflex apps compile down to traditional frontend frameworks, there's no constraint on the type of apps you can build. Instead of reinventing the web development ecosystem, we're making the existing ecosystem more accessible.
```